new/LIT6952Tasqwaqu.xml

```diff
@@ -0,0 +1,83 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6952Tasqwaqu" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ታስቈቁ፡ ጥቀ፡ ሰቃቀወ፡ ልብ፡ ወተናዝዞ፡</title>
+                <title xml:lang="gez" type="normalized" corresp="#t1">Tāsqʷaqu ṭǝqqa saqāqaw lǝbb watanāzǝzo...  (Qǝne of mawaddǝs type)</title>
```

Comment:
Tāsqʷaqqu (I think) saqāqawa…wa-tanāzǝzo

Reply:
At least Lambdin says that it is without gemination, see his 1979 book, p. 229, yāsqoqu :)